packages/components/webpack.config.js

```diff
@@ -8,13 +8,19 @@ module.exports = {
   mode: "development",
   entry: path.resolve(__dirname, "./src/index.tsx"),
   output: {
+    publicPath:
+      process.env.NODE_ENV === "production"
+        ? "https://courier-components-xvdza5.s3.amazonaws.com/"
+        : undefined,
+    filename: "latest.js",
```

Comment:
maybe will use version as the filename in the future... but for now latest until we have a discussion on versioning